README.md

```diff
@@ -1,34 +1,53 @@
 # OpenXR + Unity + Mixed Reality Samples
 
-Welcome!
+## Welcome!
 
-![OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu-Cropped.jpg)
+![OpenXR-Unity-MixedReality-Samples-MainMenu](Readme/OpenXR-Unity-MixedReality-Samples-MainMenu.jpg)
 
-These sample projects showcase features provided in the [Mixed Reality OpenXR Plugin for Unity](https://aka.ms/openxr-unity).
+These sample projects showcase how to build Unity applications 
+for HoloLens 2 or Mixed Reality headsets using the Mixed Reality OpenXR plugin.
+For more details on installing related tools and setting up a Unity project,
+please reference [the plugin documentation on https://docs.microsoft.com/](https://aka.ms/openxr-unity).
 
-Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime, which can be enabled using the OpenXR Developer Tools for Windows Mixed Reality. To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
+> NOTE: This repository uses [Git Large File Storage](https://git-lfs.github.com/) to store large files,
+such as Unity packages and images. Please install the latest git-lfs before cloning this repo.
 
-## Features showcased in the BasicSample project
+> NOTE: Features using AR Raycasts and AR Planes require the latest preview OpenXR runtime,
+which can be enabled [using the OpenXR Developer Tools for Windows Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/native/openxr-getting-started#using-preview-extensions). 
+To install this app, search for "OpenXR" in the Microsoft Store app on HoloLens 2.
 
-- **Anchors and anchor persistence** </br>
-[AnchorsSample.cs](BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs) in the ARAnchor scene demos the usage of ARFoundation to create free-world anchors, and the usage of the [XRAnchorStore](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/spatial-anchors-in-unity?tabs=openxr#using-the-anchorstore) to persist these anchors between sessions.
+### Sample for Anchors and anchor persistence
```

Comment:
nit: Sorry, I think I forgot to post this comment on the last review.
These first four "Sample for ..." headers have some inconsistent capitalization. I think sentence casing would match the rest of our documentation.
```suggestion
### Sample for anchors and anchor persistence
```